scripts/qa_crowbarsetup.sh

```diff
@@ -3810,6 +3819,33 @@ function oncontroller_testsetup()
     nova stop "$instanceid"
     wait_for 100 1 "test \"x\$(nova show \"$instanceid\" | perl -ne 'm/ status [ |]*([a-zA-Z]+)/ && print \$1')\" == xSHUTOFF" "testvm to stop"
 
+    # run tests for Magnum bay deployment
```

Comment:
You can also use the heat template created by @jgrassler here instead. https://w3.suse.de/~jgrassler/magnum-testsetup.yaml